setup.py

```diff
@@ -167,6 +174,8 @@ def build_extension(self, ext):
             cmake_args.append('-DUSE_VAI_RT=ON')
```

Comment:
We should rename this to USE_VAI_RT_DPUCADX8G and adjust other files accordingly.